<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCommandSetDialogBoxOptions</class>
 <widget class="QDialog" name="DialogCommandSetDialogBoxOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set dialog box options...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
         <item>
          <widget class="QCheckBox" name="checkBoxWindowSkinID">
           <property name="text">
            <string>Window skin ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PanelPrimitiveValue" name="panelPrimitiveWindowSkinID" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>448</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0" rowspan="2">
          <widget class="QGroupBox" name="groupBoxTransform">
           <property name="title">
            <string>Transform</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBoxX">
                <property name="text">
                 <string>X:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveX" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="2" rowspan="4">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>118</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBoxY">
                <property name="text">
                 <string>Y:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveY" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkBoxWidth">
                <property name="text">
                 <string>Width:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveWidth" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="checkBoxHeight">
                <property name="text">
                 <string>Height:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveHeight" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBoxFaceset">
           <property name="title">
            <string>Faceset</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBoxFacesetPosition">
                <property name="text">
                 <string>Position:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBoxFacesetPosition">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>118</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBoxFacesetX">
                <property name="text">
                 <string>X:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveFacesetX" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkBoxFacesetY">
                <property name="text">
                 <string>Y:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveFacesetY" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1" rowspan="3">
          <widget class="QGroupBox" name="groupBoxText">
           <property name="title">
            <string>Text</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBoxTextStroke">
                <property name="text">
                 <string>Outline:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBoxTextStroke">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="2" colspan="4">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>318</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBoxTextColor">
                <property name="text">
                 <string>Color ID:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QCheckBox" name="checkBoxTextColorText">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Text:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3" colspan="2">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveTextColorText" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="5" rowspan="3">
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>78</width>
                  <height>108</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="QCheckBox" name="checkBoxTextColorStroke">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Outline:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3" colspan="2">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveTextColorStroke" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="2">
               <widget class="QCheckBox" name="checkBoxTextColorBackground">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Background:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3" colspan="2">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveTextColorBackground" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="checkBoxTextSize">
                <property name="text">
                 <string>Size ID:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="3">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveTextSize" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="4" column="4" rowspan="2" colspan="2">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>178</width>
                  <height>48</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="0">
               <widget class="QCheckBox" name="checkBoxTextFont">
                <property name="text">
                 <string>Font ID:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="3">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveTextFont" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBoxPadding">
           <property name="title">
            <string>Padding</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBoxPaddingLeft">
                <property name="text">
                 <string>Left:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitivePaddingLeft" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBoxPaddingTop">
                <property name="text">
                 <string>Top:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitivePaddingTop" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkBoxPaddingRight">
                <property name="text">
                 <string>Right:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitivePaddingRight" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="checkBoxPaddingBottom">
                <property name="text">
                 <string>Bottom:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="PanelPrimitiveValue" name="panelPrimitivePaddingBottom" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="2" rowspan="4">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>28</width>
                  <height>118</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>318</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCommandSetDialogBoxOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCommandSetDialogBoxOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
